app/scripts/services/validateRegistrant.js

```diff
@@ -185,7 +185,8 @@ angular
       ) {
         return true;
       }
-      for (let i = 0; i < block.content.choices.length; i++) {
+
+      for (let i = 0, len = block.content.choices.length; i < len; i++) {
```

Comment:
This is a little weird looking. Guess it might slightly help performance. Keep it.

I would probably replace this whole thing with an [`Array.some()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) but that may be less performant than a `for` loop. Usually I pick the type of loop to make the code easier to read/maintain and not necessarily performance. Usually any performance difference is trivial, especially if you have a small amount of data but not always...